kuma/static/js/lang-switcher.js

```diff
@@ -62,10 +73,15 @@
 
             // Add event listener to the buttons
             $('#locale-permanent-yes').on('click', function() {
-                $.post('/i18n/setlang/', {language: this.dataset.locale})
+                var locale = this.dataset.locale;
+                $.post('/i18n/setlang/', {language: locale})
                     .success(function() {
                         notification.close();
                     });
+                // Track in GA
+                if (win.ga) {
+                    win.ga('set', 'locale-permanent-yes', locale);
```

Comment:
Let's track this as an event, like [Dashboard Pagination](https://github.com/mozilla/kuma/blob/5bbf91c2b38722c534e0aafbad0232c9898bd17d/kuma/static/js/dashboard.js#L95-L100). Here's the code for the 'yes' path:

```
mdn.analytics.trackEvent({
    category: 'Remember Language',
    action: 'yes;,
    label: locale
});
```

We'll need a ``trackEvent`` for the "no" and "never" paths as well.